api/v1alpha1/openapi.yaml

```diff
@@ -4131,6 +4191,20 @@ components:
           description: Number of restarts observed for the application.
         status:
           $ref: "#/components/schemas/ApplicationStatusType"
+        volumes:
+          type: array
+          description: Status of volumes used by this application.
```

Comment:
We could assume this as long as the application container is running. I am not sure of the value we gain here.